cfme/common/vm.py

```diff
@@ -457,6 +457,7 @@ def date_retire_element(fill_data):
 
 class VM(BaseVM):
     TO_RETIRE = None
+    VM_TYPE = None
```

Comment:
But it should be in BaseVM, not VM. Then the VM_TYPE can also hold the type of instance.